README.md

```
@@ -1,171 +1,47 @@
-# Over the Board 🏆
+# Chess Tournaments Database Archive
 
-Bem-vindo à **Over the Board**! Uma API moderna e robusta para consulta de informações sobre xadrez da Confederação Brasileira de Xadrez (CBX).
+Chess tournament data with 387995 FIDE tournaments.
 
-> **Nota**: Este é um projeto pessoal e não possui ligação oficial com a CBX.
+**This project is discontinued**
 
-## 🚀 Funcionalidades
+The data was collected by using web scraping from https://ratings.fide.com/rated_tournaments.phtml website. After [feedback](https://github.com/lichess-org/lila/issues/18989) from lichess, I've decided to discontinue at least temporary this project. The existing data is provided as-is for research, analytics, personal projects or whatever.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)